run.py

```diff
@@ -64,7 +64,7 @@
     if player.y >= 566:
         player.y = 566 
 
-    #display tank and bullets
+    #display tank, bullets and targets
```

Comment:
Thanks for updating the comments.  It's a point of conversations in both my classes, yet I still forget to check them when updating code.